pom.xml

```diff
@@ -102,8 +106,8 @@
     <dependencies>
         <dependency>
             <groupId>io.jenkins.tools.bom</groupId>
-            <artifactId>bom-2.361.x</artifactId>
-            <version>2102.v854b_fec19c92</version>
+            <artifactId>bom-${jenkins.baseline}.x</artifactId>
```

Comment:
```suggestion
            <artifactId>bom-weekly</artifactId>
```

Reply:
@alecharp Any specific reason to have `<artifactId>bom-weekly</artifactId>` (than  `<artifactId>bom-${jenkins.baseline}.x</artifactId>`) change ?

Reply:
yes, you are using a `jenkins.version` which is a weekly. So let's use the weekly BOM so that the dependencies may be updated accordingly.